cozy.example.yaml

```diff
@@ -234,6 +234,40 @@ mail:
       username: {{.Env.COZY_BETA_MAIL_USERNAME}}
       password: {{.Env.COZY_BETA_MAIL_PASSWORD}}
 
+# campaign mail service parameters for sending campaign emails via SMTP
+campaign_mail:
+  # mail smtp host - flags: --mail-host
+  host: smtp.home
+  # mail smtp port - flags: --mail-port
+  port: 587
+  # mail smtp username - flags: --mail-username
+  username: {{.Env.COZY_MAIL_USERNAME}}
+  # mail smtp password - flags: --mail-password
+  password: {{.Env.COZY_MAIL_PASSWORD}}
+  # Use SSL connection (SMTPS)
+  # Means no STARTTLS
+  # flags: --mail-use-ssl
+  use_ssl: false
+  # disable mail STARTTLS
+  # Means using plain unencrypted SMTP
+  # flags: --mail-disable-tls
+  disable_tls: false
+  # skip the certificate validation (may be useful on localhost)
+  skip_certificate_validation: false
+  # Local Name
+  # The hostname sent to the SMTP server with the HELO command
+  # Defaults to localhost
```

Comment:
Are you sure? where is this default defined? I can't find it anywhere

Reply:
I forgot to define the flags with default values. That's done now.